src/index.html

```diff
@@ -17,16 +17,27 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="big"
+      action="#"
+    >
+      <input
+        class="big form"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
```

Comment:
The class names should represent the meaning of the content, not the styles or tag names. The class 'big form' does not seem to represent the content meaningfully.